regen_go_proto_canonical_protos.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+bazel build //proto/cel/expr:all
+
+rm -vrf ./expr
+mkdir ./expr
+
+files=($(bazel aquery 'kind(proto, //proto/cel/expr:all)' | grep Outputs | grep "[.]pb[.]go" | grep "expr_go_proto" | sed 's/Outputs: \[//' | sed 's/\]//' | tr "," "\n"))
```

Comment:
nit: found more reliable way to get what we want

```suggestion
files=( $(bazel cquery //proto/cel/expr:expr_go_proto --output=starlark --starlark:expr="'\n'.join([f.path for f in target.output_groups.go_generated_srcs.to_list()])") )
```

It extracts what we want directly by using skarlark, and then printing each file on new line
```py
'\n'.join([f.path for f in target.output_groups.go_generated_srcs.to_list()])
```